public/app/features/dashboard/state/PanelModel.test.ts

```diff
@@ -388,32 +387,6 @@ describe('PanelModel', () => {
       });
     });
 
-    describe('when changing to react panel from angular panel', () => {
-      let panelQueryRunner: PanelQueryRunner;
-
-      const onPanelTypeChanged = jest.fn();
-      const reactPlugin = getPanelPlugin({ id: 'react' }).setPanelChangeHandler(
-        onPanelTypeChanged as PanelTypeChangedHandler
-      );
-
-      beforeEach(() => {
-        model.changePlugin(reactPlugin);
-        panelQueryRunner = model.getQueryRunner();
-      });
-
-      it('should call react onPanelTypeChanged', () => {
-        expect(onPanelTypeChanged.mock.calls.length).toBe(1);
-        expect(onPanelTypeChanged.mock.calls[0][1]).toBe('table');
-        expect(onPanelTypeChanged.mock.calls[0][2].angular).toBeDefined();
-      });
-
-      it('getQueryRunner() should return same instance after changing to another react panel', () => {
-        model.changePlugin(getPanelPlugin({ id: 'react2' }));
-        const sameQueryRunner = model.getQueryRunner();
-        expect(panelQueryRunner).toBe(sameQueryRunner);
-      });
-    });
-
```

Comment:
@hugohaggmark Scanning through the changes again and I'm not sure I should have deleted this. WDYT? 🤔

Reply:
Done 🙌